theforeman.org/pipelines/lib/rvm.groovy

```diff
@@ -15,6 +15,11 @@ def gemset(name = null) {
 
 def configureRVM(ruby, name = '', bundler_version = null) {
     emptyGemset(ruby, name)
+
+    if (ruby == '2.7') {
+        bundler_version = '2.4.22'
```

Comment:
Should this be `< 2.5` to allow 2.4.z patches?

Reply:
No idea. But since you are asking, I am assuming maybe yes.

Reply:
:)

I have no idea how high the chances of a 2.4.z is, but playing safe is always good :)